meetings/2021-02-09.md

```diff
@@ -4,31 +4,63 @@
 
 ### Attending
 
--
+- Audrie Fennema
+- Ross Beyer
+- Kris Becker
+- Stuart Robbins
+- Stuart Sides
+- Jesse Mapel
+- Andrew Annex
+- Lauren Adoram-Kershner
 
 ## Agenda / Notes
 
 - Action Items from last meeting
   - Jesse : Post issue about discussing significant changes to ISIS
+    - See [#133](https://github.com/USGS-Astrogeology/ISIS_TC/issues/133)
+    - Also worth looking at [#132](https://github.com/USGS-Astrogeology/ISIS_TC/issues/132)
   - Jesse/other TSC : remove extra governance text from ISIS TC app to TSC
+    - [TSC 3123](https://github.com/planetarysoftware/TSC/pull/123)
   - Jesse : send out calendar invite for next meeting
+    - Should I do this again?
+    - Yes
+    - Outlook doesn't always play nice with other calendar tools
   - Jesse : setup doodle poll about new meeting time
   - ALL : Re-read governance documents, particularly the charter
 
+- ISIS TC was accepted into the PlanetarySoftware TSC as a top level project
+
+- USGS ASC IAA projects
```

Comment:
@jlaura Can you check me on this section

Reply:
Looks good here.